src/components/SearchResultTags.js

```diff
@@ -24,12 +26,27 @@ type SearchResultTagsProps = {
    * Set to null to not show a link. Defaults to "Show 360° View."
    */
   view360Label: string | null;
+  /**
+   * If set, the search bar’s input field will use autocomplete via this URI.
+   * Otherwise, if the configuration is available, the autoCompleteUri in the
+   * configuration will be used.
+   * Otherwise, the search bar will not autocomplete.
+   * Note that this is relative to the baseUri field in the configuration.
+  */
+  autoCompleteUri: string,
+  /**
+   * Optional. The location of the node through which to interact with Attivio.
+   * Defaults to the value in the configuration.
+   */
+  baseUri: string,
```

Comment:
If `baseUri` is optional for the consumer of this component, this should be: `baseUri?: string`.

Reply:
Fixed in latest commit: https://github.com/brandon-bogan/suit/commit/9cfbb1d60f934586605da308f4e122b3766c5639